oneplus/learn_views.py

```diff
@@ -165,18 +165,10 @@ def home(request, state, user):
     redo = None
     redo_active = Setting.get_setting("REPEATING_QUESTIONS_ACTIVE")
 
-    if redo_active and redo_active == "true":
+    if redo_active and redo_active == "True":
```

Comment:
@msinovcic The Settings model in prod and QA have both been configured to use "true" to activate the feature. If you change it to "True" you will have to modify the settings in Prod and QA when this gets pushed there.